src/onboarding/Start.js

```diff
@@ -225,7 +225,11 @@ class StartContent extends React.PureComponent {
       return (
         <ActionInfo>
           You need at least {fromWei(MINIMUM_BALANCE)} ETH (you have{' '}
-          {Math.round(parseInt(fromWei(balance || '0'), 10) * 1000) / 1000}{' '}
+          {(() => {
+            const convertedBalance = parseFloat(fromWei(balance || '0'))
+            // Don't show decimals if the user has no ETH
+            return formatNumber(convertedBalance, convertedBalance ? 3 : 0)
```

Comment:
What do you think about moving this in a method, and `3` in a constant ⏫?

Reply:
I've opted to keep it scoped in `Start.js` until we start using it more: b9b6c98.